lib/games_collection.rb

```diff
@@ -14,10 +14,6 @@ def generate_objects_from_csv(csv_games_path)
   end
 
   def count_of_games_by_season
-    hash = {}
-    @games.each do |game|
-      hash[game.season] = game.game_id.length
-    end
-    hash
+    @games.map {|game| games_by_season[game.season] = game.game_id.length}
```

Comment:
I checked out your fork and noticed that game.game_id.length is calculating the length of the `game_id` string, which is ten characters long, where I think you intended to count all the unique game_id values. That's why your test's example hash comes back with exactly 10 for each season. You might hook onto some of my methods to get what you want.

I assume `games_by_season` is going to hook onto my method, so nothing to say there except I'm flattered!